### PR TITLE
[DOCFIX] Fix and update Admin-CLI docs

### PR DESCRIPTION
**Pull request by ns1123**
Add new commands and syntax
Provide example commands
@LuQQiu to update docs for 'fsadmin checkpoint' and @cheng-chang to update/link docs for 'fsadmin pathConf'.

### CODE REVIEW

File: docs/en/operation/Admin-CLI.md [hunk 1]
```diff
@@ -26,27 +27,30 @@ Usage: alluxio fsadmin [generic options]
...
-Successfully backed up journal to file:///opt/alluxio/backups/alluxio-backup-2018-5-29-1527644810.gz on master Master2
+Successfully backed up journal to /opt/alluxio/backups/alluxio-backup-2018-5-29-1527644810.gz on master Master2
 ```
 
+### checkpoint
+
 ### doctor
 
-The `doctor` command shows Alluxio errors and warnings.
+The `doctor` command gives recommendations and warnings for inconsistent configurations across different alluxio nodes.
```
Comment (LuQQiu) [inline docs/en/operation/Admin-CLI.md]: The `doctor` command is used for show Alluxio errors and warnings. Now we not only show inconsistent configuration warnings but also show lost storage warnings.

```
# shows worker lost storage warnings
./bin/alluxio fsadmin storage 
```
lost storage is the storage that worker is not able to access when starting up or running.
Like a worker has a HDD directory in its storage tier and the HDD is unmounted or is deleted. This worker cannot access the HDD directory. A warming will be shown in `fsadmin storage`

File: docs/en/operation/Admin-CLI.md [hunk 2]
```diff
@@ -60,8 +64,12 @@ It is primarily intended to assist power users in debugging their system.
 
 ```bash
 ./bin/alluxio fsadmin getBlockInfo <block_id>
+BlockInfo{id=16793993216, length=6, locations=[BlockLocation{workerId=8265394007253444396, address=WorkerNetAddress{host=local-mbp, rpcPort=29999, dataPort=29999, webPort=30000, domainSocketPath=, tieredIdentity=TieredIdentity(node=local-mbp, rack=null)}, tierAlias=MEM, mediumType=MEM}]}
+This block belongs to file {id=16810770431, path=/test2}
 ```
 
+### pathConf
```
Comment (LuQQiu) [inline docs/en/operation/Admin-CLI.md]: ?? What's that?

Comment (ns1123) [reply docs/en/operation/Admin-CLI.md]: it's a stub for the new path config feature. @cheng-chang needs to write the user docs.

File: docs/en/operation/Admin-CLI.md (hunk 1 above)
Comment (ns1123) [reply docs/en/operation/Admin-CLI.md]: got it. I'll add this to the docs

**Comment by ns1123** [comment]
jenkins, test this please

**Comment by calvinjia** [comment]
alluxio-bot, merge this please.

**Comment by calvinjia** [comment]
alluxio-bot, cherry-pick this to branch-2.0 please.

**Pull request merged**